src/hooks/useArweaveService.ts

```diff
@@ -47,38 +73,26 @@ const useArchaeologistService = () => {
     return tx.id;
   };
 
-  const updateStatus = async () => {
+  const updateStatus = async (arweareTxId: string) => {
```

Comment:
mispelling here (`arweareTxId` -> `arweaveTxId`)

Reply:
thanks, because of some light weight dyslexia, I can't even see that's misspelled.